.github/workflows/docker.yml

```diff
@@ -49,6 +49,7 @@ jobs:
             git push origin main --no-verify
             git tag -f $TAG_NAME
             git push origin --tags --no-verify
+            echo "TAG_NAME=${TAG_NAME:1}" >> $GITHUB_ENV
```

Comment:
if we want to follow semver - we should default to `1.0.0`

Reply:
We can, but I'm using raw a few lines bellow so we can keep `v1.0.0` on GH tags and `1.0.0` on dockerhub tags.